resources/views/teams/details.blade.php

```diff
@@ -4,33 +4,207 @@
+										<div class="item" data-value={{ $user->id }}>
+											<img class="ui mini circular image" src="{{ isset($user->picture_url) ? $user->picture_url : 'https://dummyimage.com/400x400/3498db/ffffff.png&text=B' }}" style="display: inline; margin-right: 10px;"/>
+											{{ $user->name }}
+										</div>
+									@endforeach
+								@endif
+							</div>
+						</div>
+						<button type="button" class="ui primary button" onclick="validateNewMember()">Add team member</button>
+					</div>
+					@if(isset($team->pending_members))
+						<div id="pendingInvites" class="{{ count($team->pending_members) <= 0 ? 'hidden' : '' }}">
+							<h2>Pending invites</h2>
+							<table class="ui striped table">
+								<tbody>
+									@foreach($team->pending_members as $pending_member)
+										<tr class="selectable">
+											<td>
+												<a href="{{ url('users', $pending_member->id) }}">
+													<img class="ui mini circular image" src="{{ isset($pending_member->picture_url) ? $pending_member->picture_url : 'https://dummyimage.com/400x400/3498db/ffffff.png&text=B' }}" style="display: inline; margin-right: 10px;"/>
+													{{ $pending_member->name }}
+												</a>
+											</td>
+										</tr>
+									@endforeach
+								</tbody>
+							</table>
+						</div>
+					@endif
+					<div id="member-to-add-modal" class="ui modal">
+						<p class="header">Add member</p>
+						<div class="content">
+							<p>Invite</p>
+							<p><strong id="newMemberName"></strong></p>
+							<p>to join <strong>{{ $team->name }}.</strong></p>
+						</div>
+						<div class="actions">
+							<button type="button" class="ui cancel button">Cancel</button>
+							<button type="button" class="ui ok primary button" onclick="inviteMember()">Confirm</button>
+						</div>
+					</div>
 				@endif
 			</div>
-		</div>
-	</div>
-	<div class="team-projects">
-		<p><strong>Projects</strong></p>
-		<!-- TODO: add project cards -->
-		<div class="ui message">
-			<div class="header"> No projects available </div>
-			<p>Looks like this team has no projects at the moment.</p>
 		</div>
 	</div>
 </div>
+@section("scripts")
+<script src="{{ mix('js/utilities.js')}}"></script>
+<script>
+	/* Semantic UI tabs */
+	$(".menu .item").tab();
+	$("#member-to-add-modal").modal({ transition: "fade up" });
+
+	@if(Auth::id() === $team->leader->id)
+		/* Semantic UI dropdown */
+		$("#new-member-dropdown").dropdown({
+			onChange: function() {
+				$("#new-member-dropdown").removeClass("error");
+			}
+		});
+
+		/* render sent datetime for all invites*/
+		renderDateTimeAgoOnce();
+
+		/* Validate invitation to join the team */
+		function validateNewMember() {
+			// get the info of the user that will receive the invitation
+			const newMemberName = $("#new-member-dropdown").dropdown("get text");
+			const newMemberId = $("#new-member-dropdown").dropdown("get value");
+
+			if(newMemberId === $("#new-member-dropdown").dropdown("get default value")) {
```

Comment:
El default value tmb deber&iacute;as separarle en su propia variable, this looks very ugly